### PR TITLE
Feature/dx 98 create glossary list

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: resources/terminology.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+| **Verify**     | The payment operation that initiates a [verification][verify-url] payment process. It is a payment that lets you post verifications to confirm the validity of card information, without reserving or charging any amount. This option is used to generate a payment- or recurrence token, that can be used in a recurring payments scenarios or for one-clickpayments, without charging the card in the process. |
+
+[admin-manual]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
+[admin-portal]: https://admin.externalintegration.payex.com/psp/login
+[admin-getting-started]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
```
Comment (NullableInt) [inline resources/terminology.md]: We don't need to link to info about the "old" admin in the old developer portal pages.

File: resources/terminology.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      title: Open Source Development Guidelines
+    - url: /resources/terminology
+      title: Terminology
+---
+
+{% include alert-development-section.md %}
+
+{:.table .table-striped}
+| Term  | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| :------ | :------------ |
```
Comment (NullableInt) [inline resources/terminology.md]: Auto format table `Alt + Shit + F`

File: resources/terminology.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+| **Account Receivable Consumer**                 | The [`AccountReceivableConsumer`][invoice-url] API is the fundament for Swedbank Pay [Invoice Payments][invoice-url] service. It is a service where Swedbank Pay helps produce and distribute invoices to consumers/end-users.                                                                                                                                                                                                                                               |
```
Comment (NullableInt) [inline resources/terminology.md]: It's a bit odd to have two identical links in the same sentence.

File: resources/terminology.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+[admin-manual]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
+[admin-portal]: https://admin.externalintegration.payex.com/psp/login
+[admin-getting-started]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
+[admin-interface-search]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin-account-interface-and-search/
+[auto-capture]: /payments/card/other-features#capture
+[hateoas]: https://en.wikipedia.org/wiki/HATEOAS
+[invoice-url]: /payments/invoice/
+[two-phase-payments]: /payments/#the-fundamentals
+[callback-url]:/payments/card/other-features#callback
+[cancel-url]: /checkout/after-payment#cancel
```
Comment (NullableInt) [inline resources/terminology.md]: Why is this the only link that goes to `/checkout` when the other links to specific pages go to `/payments/card`?

File: resources/terminology.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+[checkout-url]: payments/checkout/
```
Comment (NullableInt) [inline resources/terminology.md]: This link does not work.

File: resources/terminology.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+[common-headers]: payments/#headers
```
Comment (NullableInt) [inline resources/terminology.md]: This link does not work.

File: resources/terminology.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+[restful-api]: payments/#connection-and-protocol
+[verify-url]: /payments/card/other-features#verify
+[one-phase-url]: /payments/#the-fundamentals
+[payout]: /payments/card/other-features#payout
+[recur]: /payments/card/other-features#recur
+[sale-authorization-url]: /payments/#creating-an-authorization-or-sales-transaction
```
Comment (NullableInt) [inline resources/terminology.md]: This link does not work.

File: resources/terminology.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+| **Payee**                     | The company that receive funds for the payment.                                                                                                                                                                                                                                                                                                                                                                     |
+| **Payee ID**                 | The ID of the company that receive funds for the payment. May also be known as Merchant ID.                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (NullableInt) [inline resources/terminology.md]:
```suggestion
| **Payee ID**                 | The ID of the company that receive funds for the payment. Also referred to as Merchant ID.                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

File: resources/terminology.md (hunk 4 above)
Comment (marinafrydenb) [reply resources/terminology.md]: changed

File: resources/terminology.md (hunk 6 above)
Comment (marinafrydenb) [reply resources/terminology.md]: fixed

File: resources/terminology.md (hunk 5 above)
Comment (marinafrydenb) [reply resources/terminology.md]: fixed

File: resources/terminology.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+| **Account Receivable Consumer** | The [`AccountReceivableConsumer`][invoice-url] API is the fundament for Swedbank Pay [Invoice Payments][invoice-url] service. It is a service where Swedbank Pay helps produce and distribute invoices to consumers/end-users.                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (asbjornu) [inline resources/terminology.md]: It's not obvious from this description how `AccountReceivableConsumer` is different to `FinancingConsumer`. Both of these concepts should be more thorougly explained in Invoice Payments Other Features. Can you add a task for it?

File: resources/terminology.md (hunk 7 above)
Comment (marinafrydenb) [reply resources/terminology.md]: fixed

File: resources/terminology.md (hunk 9 above)
Comment (marinafrydenb) [reply resources/terminology.md]: Yes, creating a task for 1) and 2) and invoice payments other features 3)

**Pull request merged**